app/services/comment_service.rb

```diff
@@ -50,12 +50,15 @@ def self.destroy(comment:, actor:)
 
   def self.update(comment:, params:, actor:)
     comment.edited_at = Time.zone.now
+    new_mentions = User.where username: Comment.new(params).mentioned_usernames - comment.mentioned_usernames
```

Comment:
Think I prefer instantiating a new comment to having the CommentService know what a Twitter::Extractor is.

Also, maybe I should throw an empty check in there so we're not hitting the DB if there's not new mentions.